src/tests/build_test.ts

```diff
@@ -56,192 +55,5 @@ Deno.test("build hook tests", async (t) => {
         runStub.restore();
       },
     );
-
```

Comment:
Essentially all of these tests are now moved to `src/tests/utilities_tes.ts`, since the logic for this kind of function validation exists in there, since it is re-used across multiple hooks.